src/Migration.ps1

```diff
@@ -214,18 +228,77 @@ function Main
     	$logger.LogInfoObject("Not Migrated AMSv1 Unsupported Provider list - ", $unsupportedProviderList);
 	}
 
+	$compareLaws = Get-CompareLaws -amsv1ArmId $amsv1ArmId -amsv2ArmId $amsv2ArmId -logger $logger
+	$isMigrateAlerts = "";
+
+	if($compareLaws.isEqual -eq $true)
+	{
+		$logger.LogInfo("AMSv1 and AMSv2 LAWS - $($compareLaws.amsv2LawsId) are equal..");
+	}
+	else 
+	{
+		$logger.LogInfo("AMSv1 $($compareLaws.amsv1LawsId) and AMSv2 $($compareLaws.amsv2LawsId) LAWS are NOT equal..");
+		[string]$dialogContent = "Please select an action for Alert Migration.";
+		Write-Host $dialogContent;
+		while (($isMigrateAlerts -ne "yes") -and ($isMigrateAlerts -ne "no")) {
+			$isMigrateAlerts = Read-Host -Prompt "Do you wish to Migrate Alerts? (yes/no)";
+		}
+	}
+
+	if($isMigrateAlerts -like "yes") {
```

Comment:
So for no there is no particular case to handle, only this if check for "yes" and the corresponding action suffices ?

Reply:
maybe we can add a log. other than that no action required i believe

Reply:
No need for logs, I was just clarifying.

Reply:
Updated small log for info.